### ISSUE TITLE
Is the app now completely offline?

### ISSUE THREAD

**Issue by Hubgut** [SecUSo/privacy-friendly-weather#147]
I personally wasn't aware of this until yesterday evening since I had no reason to look for the weather forecast that often (with all this sitting at home during the pandemic). But yesterday someone I recommended the app to, ask me about the impossibility of updating the weather data anymore.
So I looked at my installation and saw I was also unable to get any more data. Then I looked at the developers page and installed the newest version (before 2.4 I had 2.2 installed) – but this one also never got any data!?

So, is the app now completely unusable for normal users, without an own API Key?
Or is it currently blocked from the used API?

I don’t care for high refresh rates but since my update, more than an hour ago, I did not get a single piece of data, not a forecast for one of my cities, not any content for one of the widgets.
I don’t know how long this was the situation of the version 2.2, I had installed before.

I also would expect some tutorial or help for the more technical users, some How-To get your own API Key, maybe here in the wiki.

Apart from that I would strongly favour a multiple-key-solution with shifting usage within the app for the normal users. Otherwise the app is looking quite dead right now. --- It would be a shame if it died, because “too many users did use it”!

**Comment by sleep-yearning** [comment]
Yes, the API key is currently "run out for the day" most of the time.
We are aware of the problem and will have to solve it one or the other way.

**Comment by Hubgut** [comment]
So, I did **not get a single piece of data since** I freshly installed the update, **two days ago**. This will definitively look like a ***broken app for every normal user***!
In my opinion, it's not a fine idea to publish the app on Google playstore, until this is fixed. This will create bad publicity! Their users most likely won't look into creating their own API Key.

I was advertising this app for friends and family for some time now. Now I had to create an own API Key, just to see if the app is working any more. I will try to communicate this to others, but I doubt that they will understand nor respect this. --- Most users does not know anything besides the Playstore App. It's most unlikely that anybody will invest this (quite little) effort to get and copy in their own API Key. These are plain install-and-use-users nowadays.
